var/spack/repos/builtin/packages/umap/package.py

```diff
@@ -3,7 +3,9 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
+import os
+
+from spack.error import UnsupportedPlatformError
```

Comment:
Although you can definitely import specific error messages like this, I'd also recommend keeping the top level `from spack.package import *` directive in the package for future maintainers/to keep the package consistent with the styling of other packages.